README.md

```diff
@@ -30,3 +30,18 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+T(n) { 1 for n <= 1 <br>
+       3T(n/3) + n<sup>5</sup> for n > 1 <br>
+       - 3 recursive calls within the function, each called with n/3 <br>
+       - triple nested loop, first loops from 0 to n<sup>2</sup>, second from 0 to n, third from 0 to n<sup>2</sup>, with a constant time calculation done inside. Thus giving us n^5 time. <br>
+       - 3(3T(n/3/3) + (n/3)<sup>5</sup>) + n<sup>5</sup> <br>
+       - 9T(n/9) + n<sup>5</sup>/3<sup>5</sup> + n<sup>5</sup> <br>
+       - 3<sup>i</sup>T(n/3<sup>i</sup>) + n<sup>5</sup>/3<sup>5i</sup> + ... + n<sup>5</sup> <br>
+       - find base case/termination condition in terms of i <br>
+       - n/3<sup>i</sup> = 1, i = log<sub>3</sub>(n) <br>
+       - 3<sup>log<sub>3</sub>(n)</sup>T(n/3<sup>log<sub>3</sub>(n)</sup>) + ... + n<sup>5</sup> <br>
+       - n * 1 + ... + n<sup>5</sup> <br>
+       - lower order terms ignored <br>
```

Comment:
But they are all $n^5$ terms, not lower order.

Reply:
My base level knowledge and thinking was that since all the following calls add on an n^5 term decreasing by a factor of 1/3^5x (with x being the level of recursion) then the sum of them all would still be some multiple of n^5 and simplify to an asymptotic complexity of O(n^5).

Reply:
That sounds like it's going in the right direction -- can you please clarify that in your submission?